api_tests/nodes/views/test_node_alternative_citations.py

```diff
@@ -0,0 +1,1098 @@
+        self.private_url = '/{}nodes/{}/citations/'.format(API_BASE, self.private._id)
+        self.public_url_bad = self.public_url + '1/'
+        self.private_url_bad = self.private_url + '1/'
+        self.citation.public_url = self.public_url + '{}/'.format(self.citation._id)
+        self.citation.private_url = self.private_url + '{}/'.format(self.citation._id)
+
+    def test_delete_citation_admin_public(self):
+        res = self.app.delete_json_api(self.citation.public_url, auth=self.user.auth)
+        assert_equal(res.status_code, 204)
+        self.public.reload()
+        assert_equal(len(self.public.alternativeCitations), 0)
+
+    def test_delete_citation_admin_private(self):
+        res = self.app.delete_json_api(self.citation.private_url, auth=self.user.auth)
+        assert_equal(res.status_code, 204)
+        self.private.reload()
+        assert_equal(len(self.private.alternativeCitations), 0)
+
+    def test_delete_citation_non_admin_public(self):
+        res = self.app.delete_json_api(self.citation.public_url, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.public.reload()
+        assert_equal(len(self.public.alternativeCitations), 1)
+
+    def test_delete_citation_non_admin_private(self):
+        res = self.app.delete_json_api(self.citation.private_url, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.private.reload()
+        assert_equal(len(self.private.alternativeCitations), 1)
+
+    def test_delete_citation_non_contrib_public(self):
+        res = self.app.delete_json_api(self.citation.public_url, auth=self.user_three.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.public.reload()
+        assert_equal(len(self.public.alternativeCitations), 1)
+
+    def test_delete_citation_non_contrib_private(self):
+        res = self.app.delete_json_api(self.citation.private_url, auth=self.user_three.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.private.reload()
+        assert_equal(len(self.private.alternativeCitations), 1)
+
+    def test_delete_citation_admin_404_public(self):
+        res = self.app.delete_json_api(self.public_url_bad, auth=self.user.auth, expect_errors=True)
+        assert_equal(res.status_code, 404)
+        self.public.reload()
+        assert_equal(len(self.public.alternativeCitations), 1)
+
+    def test_delete_citation_admin_404_private(self):
+        res = self.app.delete_json_api(self.private_url_bad, auth=self.user.auth, expect_errors=True)
+        assert_equal(res.status_code, 404)
+        self.private.reload()
+        assert_equal(len(self.private.alternativeCitations), 1)
+
+    def test_delete_citation_non_admin_404_public(self):
+        res = self.app.delete_json_api(self.public_url_bad, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
```

Comment:
Some of these tests have "404" in the test name, but return `403` errors. Can you change the name of these tests?